game/graphics/pipelines/opengl.cpp

```diff
@@ -389,12 +402,24 @@ void GLDisplay::on_iconify(GLFWwindow* /*window*/, int iconified) {
 }
 
 namespace {
-std::string make_output_file_name(const std::string& file_name) {
+std::string make_full_screenshot_output_file_path(const std::string& file_name) {
   file_util::create_dir_if_needed(file_util::get_file_path({"gfx_dumps"}));
   return file_util::get_file_path({"gfx_dumps", file_name});
```

Comment:
we can probably rename this folder to `screenshots` now.

Reply:
Done